packages/super-editor/src/extensions/table/tableHelpers/createTable.js

```diff
@@ -12,13 +12,16 @@ import { createTableBorders } from './createTableBorders.js';
  * @param {number} colsCount - Number of columns
  * @param {boolean} withHeaderRow - Create first row as header
  * @param {Object} [cellContent=null] - Initial cell content
+ * @param {number[]} [columnWidths=null] - Array of pixel widths per column
  * @returns {Object} Complete table node with borders
  * @example
  * const table = createTable(schema, 3, 3, true)
  * @example
  * const table = createTable(schema, 2, 4, false, paragraphNode)
+ * @example
+ * const table = createTable(schema, 3, 3, false, null, [200, 100, 200])
  */
-export const createTable = (schema, rowsCount, colsCount, withHeaderRow, cellContent = null) => {
+export const createTable = (schema, rowsCount, colsCount, withHeaderRow, cellContent = null, columnWidths = null) => {
```

Comment:
Recommend changing function signature to take an object with named options, rather than an ordered list of positional arguments; with a long list of arguments, it's hard to look at a `createTable` call and understand what each argument is.